### PR TITLE
Interpretable Elo scores

### LINKED ISSUE #113: Output estimated score on interpretable Elo scale
Currently the tuner outputs the "estimated value" which is the negative Elo score divided by 100. To users of the tuning tools this is very confusing and something easy to fix just for the output.

### PR DESCRIPTION
**Pull request by Eugenio-Bruno**
This (seems to) partially address the score interpretability problem, by printing the Elo equivalent to the score.

To be checked: whether I've scaled stuff correctly
Still to be done: make the charts use the same scale
Maybe to be done: document the change? This might make things *more* confusing to existing users of the tool.

It also seems like the commit history is messed up, so maybe if you merge just squash it? Not sure how to deal with it.

Fixes #113

### CODE REVIEW

**Comment by Eugenio-Bruno** [comment]
Should mention it's about https://github.com/kiudee/chess-tuning-tools/issues/113.

File: tune/cli.py [hunk 1]
```diff
@@ -371,15 +371,15 @@ def local(  # noqa: C901
...
                 root_logger.info(
-                    f"Estimated value: {np.around(best_value, 4)} +- "
-                    f"{np.around(best_std, 4).item()}"
+                    f"Estimated Elo: {np.around(-best_value * 100, 4)} +- "
+                    f"{np.around(best_std * 100, 4).item()}"
                 )
                 confidence_val = settings.get("confidence", confidence)
                 confidence_mult = erfinv(confidence_val) * np.sqrt(2)
                 root_logger.info(
                     f"{confidence_val * 100}% confidence interval of the value: "
```
Comment (kiudee) [inline tune/cli.py:380]: Also change `value` to `Elo score` or `Elo value`

**Review by kiudee — CHANGES_REQUESTED** [review]
Other than the very minor output change this looks good to me.
I will add the change to the changelog after it is merged.

File: tune/cli.py (hunk 1 above)
Comment (Eugenio-Bruno) [reply tune/cli.py]: I *think* I've changed what you asked. I used `Elo value` to be consistent with the internal naming.

**Pull request merged**